.github/workflows/post-release-changes.yaml

```diff
@@ -0,0 +1,51 @@
+name: Post-release changes
+
+on:
+  release:
+    types: [published]
+
+env:
+  PYTHON_VERSION: 3.9
+
+jobs:
+  post-release-changes:
+    runs-on: ubuntu-latest
+    steps:
+    # setup pre-reqs
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0 # fetch all commits/branches
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ env.PYTHON_VERSION }}      
+    - name: Get version
+      run: |
+        tarref=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+        if [[ "${{ github.ref }}" == "refs/tags/"* ]]; then
+          # Strip "v" prefix from tagref
+          echo "VERSION=$(echo $tarref | sed -e 's/^v//')" >> $GITHUB_ENV
+          echo "MAJOR_MINOR_VERSION=$(echo $tarref | sed -e 's/^v//' -e 's,\([0-9]*\.[0-9]*\)\.\([0-9]*\),\1,')" >> $GITHUB_ENV
+        elif [[ "${{ github.ref }}" == *"master" ]]; then
```

Comment:
We should ensure that `major_minor_version` is properly extracted here always... if not, the workflow can fail without doing anything else.

Please see earlier comment on semantic versioning.

Reply:
I added an additional step where we run the version number through a semver Python module. I believe this concern has been resolved.